locobot/agent/locobot_mover.py

```diff
@@ -244,3 +244,20 @@ def explore(self):
 
     def drop(self):
         return self.bot.open_gripper()
+
+    def get_obstacles_in_canonical_coords(self):
+        """get the positions of obtacles position in the canonical coordinate system
+        instead of the Locobot's global coordinates as stated in the Locobot
+        documentation: https://www.pyrobot.org/docs/navigation.
```

Comment:
nit: the pyrobot site has been down for quite some time. Is there maybe a sphinx docs link for this? some place that is not dead.

Reply:
added the replacement [link](https://github.com/facebookresearch/pyrobot/blob/master/docs/website/docs/ex_navigation.md)